BUILDING.md

```diff
@@ -1,126 +1,152 @@
+- **Debian/Ubuntu:**
 
-### Tek parça HTML olarak inşa etme
+  ```shell
+  make singlehtml
+  ```
 
-Debian/Ubuntu'da:
+- **Windows:**
 
-```shell
-make singlehtml
-```
+  ```shell
+  make.bat singlehtml
+  ```
 
-Windows'ta:
+  Oluşturulan HTML dosyası `/build/singlehtml/index.html` yolunda bulunacaktır.
 
-```shell
-make.bat singlehtml
-```
+---
 
-HTML dosyası `/build/singlehtml/` dizininde `index.html` adı ile oluşacaktır.
+### EPUB
 
-### EPUB olarak inşa etme
+- **Debian/Ubuntu:**
 
-Debian/Ubuntu'da:
+  ```shell
+  make epub
+  ```
 
-```shell
-make epub
-```
+- **Windows:**
 
-Windows'ta:
+  ```shell
+  make.bat epub
+  ```
 
-```shell
-make.bat epub
-```
+  EPUB dosyası `/build/epub/Yazbel Python Belgeleri.epub` olarak kaydedilecektir.
 
-EPUB dosyası `/build/epub/` dizininde `Yazbel Python Belgeleri.epub` adı ile oluşacaktır.
+---
 
-### PDF olarak inşa etme
+### PDF
 
-Belgeleri PDF olarak inşa edebilmek için ``pdflatex`` uygulamasına ihtiyacınız olacak. [MikTeX](https://miktex.org/) veya [TeX Live](https://www.tug.org/texlive/) gibi bir TeX dağıtımını indirerek bu uygulamayı edinebilirsiniz. Bu dağıtımların belgerin inşası için gerekli eklentiler ile birlikte 800 Megabyte gibi bir disk alanı kaplayabileceğini unutmayın. TeX dağıtımının kurulumunda bir problem yaşarsanız [buraya](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.latex.LaTeXBuilder) başvurabilirsiniz.
+PDF formatında belgeleri oluşturmak için bir TeX dağıtımı (ör. [MikTeX](https://miktex.org/) veya [TeX Live](https://www.tug.org/texlive/)) kurmanız gereklidir. Bu dağıtımlar genellikle ek eklentilerle birlikte kurulur ve yaklaşık 800 MB disk alanı gerektirir.
 
-> Eğer Windows kullanıyorsanız ve [`winget`](https://github.com/microsoft/winget-cli) CLI uygulamasına sahipseniz MikTeX dağıtımını indirmek için bu yolu da izleyebilirsiniz:
+> **Windows Kullanıcıları için:** Eğer [`winget`](https://github.com/microsoft/winget-cli) kuruluysa, MikTeX'i şu komutlarla yükleyebilirsiniz:
 >
 > ```shell
 > winget install MiKTeX.MiKTeX
-> winget install StrawberryPerl.StrawberryPerl # MiKTeX aynı zamanda bir Perl kurulumu gerektirir
+> winget install StrawberryPerl.StrawberryPerl # MikTeX ayrıca bir Perl kurulumu gerektirir
 > ```
 
-Uygun bir TeX dağıtımını kurduktan sonra `pdflatex`'in bulunduğu dizinin PATH'de bulunduğundan emin olun.
+TeX dağıtımı kurulduktan sonra `pdflatex` uygulamasının PATH ortam değişkeninde yer aldığından emin olun.
+
+Belgeleri PDF formatında oluşturmak için:
+
+- **Debian/Ubuntu:**
 
-Debian/Ubuntu'da:
+  ```shell
+  make latexpdf
+  ```
 
-```shell
-make latexpdf
-```
+- **Windows:**
 
-Windows'ta:
+  ```shell
+  make.bat latexpdf
+  ```
 
-```shell
-make.bat latexpdf
-```
+  Oluşturulan PDF dosyası `/build/latex/yazbelpythonbelgeleri.pdf` yolunda bulunacaktır.
 
-Herhangi bir hata oluşmazsa PDF dosyası `/build/latex/` dizininde `yazbelpythonbelgeleri.pdf` adı ile oluşacaktır.
```

Comment:
Normalde bir aksaklık olursa kişi adımları doğru takip etmediğinden şüphelenir. Ama PDF inşası sıkıntılı bir süreç, bu adımlar her senaryoyu karşılamıyor da olabilir. Buradaki vurgunun sebebi bu manayı iletmekti aslında.

"PDF inşası sürecinde sorun yaşarsanız repo'da issue açabilirsiniz" diye ekleyelim mi?